cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/CoreDatasetsModule.java

```diff
@@ -67,7 +66,6 @@ public void register(DatasetDefinitionRegistry registry) {
     registry.add(new CounterTimeseriesTableDefinition(CounterTimeseriesTable.class.getName(), tableDef));
 
     // in-memory table
-    registry.add(new InMemoryTableDefinition(MemoryTable.class.getName()));
```

Comment:
There is no class implementing `MemoryTable` interface, hence this registration is useless.